sambacc/samba_cmds.py

```diff
@@ -159,19 +159,19 @@ def __repr__(self) -> str:
 samba_dc = SambaCommand("/usr/sbin/samba")
 
 
-def smbd_foreground():
+def smbd_foreground() -> SambaCommand:
     return smbd[
         "--foreground", _daemon_stdout_opt("smbd"), "--no-process-group"
     ]
```

Comment:
IMHO, now that we have explicit return type, those lines look a bit enigmatic. I would consider (in a separate PR) to have more explicit code (also in other places below):

```
def smbd_foreground() -> SambaCommand:
    return SambaCommand("/usr/sbin/smbd",
        [ "--foreground", _daemon_stdout_opt("smbd"), "--no-process-group" ])
```